style.css

```diff
@@ -0,0 +1,186 @@
+/* stylelint-disable */
+
+
+/* GLOBALS */
+
+html {
+    box-sizing: border-box;
+    font-size: 62.5%;
+    /* Rem Reset */
+    font-family: 'Lato', sans-serif;
+}
+
+*,
+*::before,
+*::after {
+    box-sizing: inherit;
+}
+
+img {
+    max-width: 100%;
+    height: auto;
+}
+
+.d-flex {
+    display: flex;
+}
+
+.center-flex {
+    justify-content: space-around;
+    align-items: center;
+}
+
+.center-container {
+    margin: 0 auto;
+}
+
+.center-text {
+    text-align: center;
+}
+
+
+/*---------------------*/
+
+
+/* Background (Header) */
+
+.hero-container {
+    position: relative;
+    height: 80.5vh;
+}
+
+.hero {
+    position: absolute;
+    top: 0;
+    background-image: url(images/your_verse_hero_2x.jpg);
+    background-size: cover;
+    background-position: center bottom;
+    background-repeat: no-repeat;
+    height: 80vh;
+    min-height: 40rem;
+    min-width: 80rem;
+}
+
+
+/*------------------*/
+
+
+/* navigation bar */
+
+.nav-container {
+    border: 1px solid #646764;
+    border-radius: 0.3rem;
+    width: 75vw;
+    margin: 5rem auto 0;
+    background: linear-gradient(0deg, #6c6c6c, #505050 50%, #646764 50%, #848484 100%);
+}
```

Comment:
- [ ] The first and last elements in your navbar are bigger than the rest, you can fix the issue just by giving a 'fixed' width to it. **Hint:** use rem units and see the difference :) 
Your navbar:
![image](https://user-images.githubusercontent.com/53256139/91106504-7729b300-e638-11ea-9d4d-364017a75bc6.png)